pkg/utils/reconciler.go

```diff
@@ -32,11 +32,13 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client/apiutil"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
-	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"sigs.k8s.io/yaml"
 )
 
+var	logger = NewLogger(true) //log in JSON format
```

Comment:
How does this logger get the logging setting from the kAppNav CR? I don't see it being applied.